code/game/objects/items/tanks/tank_types.dm

```diff
@@ -181,3 +179,26 @@
 	name = "double emergency oxygen tank"
 	icon_state = "emergency_engi"
 	volume = 10
+
+/*
+ * Nitrogen Tanks
+ */
+
+/obj/item/tank/internals/emergency_oxygen/vox
+	name = "vox n2 tank"
+	desc = "A vox nitrogen tank made of a strong, light weight alloy; this tank is capable of holding gasses at high pressures without exploding. Due to it's light weight, don't expect to robust any dustlungs with it, skrek."
+	icon_state = "emergency_vox"
+	flags_1 = CONDUCT_1
+	slot_flags = SLOT_BELT
+	w_class = WEIGHT_CLASS_SMALL
+	force = 0 // No, you don't get to break out of the brig using your voxygen tank, shitbird.
+	distribute_pressure = TANK_DEFAULT_RELEASE_PRESSURE
+	volume = 8 // Larger than a standard emergency tank, smaller than a double.
+	attack_verb = list("shitbirded")

```

Comment:
too OOC. Make it things like "skrekked" or "skreed" or something.

Reply:
Skrekked is for the tongue. I'll figure it out, though. Hmm.

Reply:
is it really too OOC? i call em shitbirds ICly